server/server.js

```diff
@@ -17,8 +17,7 @@ var errorHandler = require('strong-error-handler');
 
 // required to support base models
 app.dataSource('db', {
-  connector: loopback.Memory,
-  defaultForType: 'db',
+  connector: "memory"
```

Comment:
I think this change should not be necessary, but it should not break anything either, so I'm fine with keeping it :+1: